_metadata.py

```diff
@@ -1,2 +1,2 @@
-__extension_version__ = "0.6.0"
+__extension_version__ = "0.6.1"
```

Comment:
With an update of the qujax version I would change to 0.7.0
```suggestion
__extension_version__ = "0.7.0"
```

Reply:
ok!